style.css

```diff
@@ -124,7 +125,11 @@ body
     /*horizontal(offset x) zero vertical(offset y) zero blur radius zero and spread radius set to 1px*/
     box-shadow: inset 0 0 0 1px #fff;
   }
-}
+  .hamburger-icon-mobile
+  {
+  display: none;
+  }
+  }
```

Comment:
New line between each class

Reply:
Is this fixed? I don't see it.

Reply:
Its in navbar branch right.I pushed that changes there.